### PR TITLE
updates tooling choices

### LINKED ISSUE #20: Update tooling options to add supported tools
As a user, I want the tooling options updated in the database. Supported tools are: 

* GitHub
* TravisCI
* Docker
* Heroku
* Mocha
* Sequelize

### PR DESCRIPTION
**Pull request by jackcmeyer**
closes #20 

* updates tooling choices and makes it consistent with tyrfiles.

### CODE REVIEW

File: src/db/init_database.js [hunk 1]
```diff
@@ -18,11 +27,8 @@ export async function populateTools() {
     specialConsiderations: 'To use the open source version of TravisCI, you must have a GitHub account.'
   });
   await sequelize.Tool.create({
-    name: 'Docker Hub',
+    name: 'Docker',
     toolType: sequelize.ToolType.CONTAINERIZATION,
-    websiteUrl: 'https://hub.docker.com/',
```
Comment (KrashLeviathan) [inline src/db/init_database.js]: Instead of removing, change to `https://www.docker.com/`

File: src/db/init_database.js [hunk 2]
```diff
@@ -18,11 +27,8 @@ export async function populateTools() {
...
-    apiUrl: 'https://index.docker.io/v1/',
-    documentationUrl: 'https://docs.docker.com/',
```
Comment (KrashLeviathan) [inline src/db/init_database.js]: Keep this one in.

File: src/db/init_database.js [hunk 3]
```diff
@@ -45,6 +51,20 @@ export async function populateTools() {
     documentationUrl: 'http://expressjs.com/en/4x/api.html',
     logoLargeUrl: 'https://camo.githubusercontent.com/fc61dcbdb7a6e49d3adecc12194b24ab20dfa25b/68747470733a2f2f692e636c6f756475702e636f6d2f7a6659366c4c376546612d3330303078333030302e706e67',
   });
+  await sequelize.Tool.create({
+    name: 'Mocha',
+    toolType: sequelize.ToolType.TEST,
+    websiteUrl: 'https://mochajs.org/',
+    documentationUrl: 'https://mochajs.org/#getting-started',
+    logoLargeUrl: 'https://worldvectorlogo.com/logo/mocha-1'
```
Comment (KrashLeviathan) [inline src/db/init_database.js]: This should be a link to an actual image file, not a webpage holding an image file. That way we can easily insert the images wherever we want them.

File: src/db/init_database.js [hunk 4]
```diff
@@ -45,6 +51,20 @@ export async function populateTools() {
...
+  });
+  await sequelize.Tool.create({
+    name: 'Sequelize',
+    toolType: sequelize.ToolType.TEST,
```
Comment (KrashLeviathan) [inline src/db/init_database.js]: I think you said you were going to fix the toolType for this one, but I'm adding a comment to remind you.

**Comment by jackcmeyer** [comment]
@KrashLeviathan closing this PR as #25 better addresses it

**Pull request closed without merging**